src/apps/orders/services/order_refunder.py

```diff
@@ -109,11 +146,16 @@ def get_email_template_context(self) -> dict:
         return {
             "order_id": self.order.pk,
             "refunded_item": self.order.item.name if self.order.item else "not-set",
-            "refund_author": str(get_current_user()),
+            "refund_author": str(self.refund_author),
             "payment_method_name": self.payment_method_before_service_call,
             "price": format_price(self.order.price),
+            "amount": format_price(self.amount),
             "order_admin_site_url": urljoin(
                 settings.ABSOLUTE_HOST,
                 reverse("admin:orders_order_change", args=[self.order.pk]),
             ),
         }
+
+    def update_price(self) -> None:
+        self.order.price = self.order.price - self.amount
+        self.order.save(update_fields=["price"])
```

Comment:
```suggestion
        self.order.save(update_fields=["modified", "price"])
```
Кажется иначе мы потеряем обновление поля modified.